src/recommendation/src/main/python/SAR.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for information.
+
+
+import sys
+
+if sys.version >= '3':
+    basestring = str
+
+from pyspark.ml.param.shared import *
+from pyspark.ml.common import inherit_doc
+from mmlspark.Utils import *
+from mmlspark._SAR import _SAR as sar
+from mmlspark.SARModel import SARModel as sarm
+
+
+@inherit_doc
+class SAR(sar):
+    def _create_model(self, java_model):
```

Comment:
because you took out the json encoding and decoding of parameters, you might need to add some wrapper code here to convert to and prom python dataframes.